algoliasearch/errors.go

```diff
@@ -10,6 +10,8 @@ var (
 	NoMoreSynonymsErr           error = errors.New("No more synonyms")
 	NoMoreRulesErr              error = errors.New("No more rules")
 	ExhaustionOfTryableHostsErr error = errors.New("All hosts have been contacted unsuccessfully")
+	SameAppIDErr                error = errors.New("Application IDs cannot be identical")
+	IndexAlreadyExistsErr       error = errors.New("Index already exists")
```

Comment:
"Destination index already exists. Please delete it before copying index across applications."